src/FTPC.jl

```diff
@@ -266,6 +263,13 @@ function ftp_get(ctxt::ConnContext, file_name::AbstractString, save_path::Abstra
         p_wd = pointer_from_objref(wd)
         p_resp = pointer_from_objref(resp)
 
+        @ce_curl curl_easy_setopt CURLOPT_VERBOSE Int64(1)
+
+        # Force active mode
+        @ce_curl curl_easy_setopt CURLOPT_FTP_USE_EPSV 0
+        @ce_curl curl_easy_setopt CURLOPT_FTP_USE_EPRT 0
+        @ce_curl curl_easy_setopt CURLOPT_FTPPORT "-"
```

Comment:
Verbose option was just meant for debugging and should be removed. Comment out the "force active mode" code as it was just meant for testing. We probably want to support that functionality later.